data/basal.go

```diff
@@ -0,0 +1,166 @@
+package data
+
+import (
+	"fmt"
+	"reflect"
+	"strings"
+
+	valid "github.com/tidepool-org/platform/Godeps/_workspace/src/gopkg.in/bluesuncorp/validator.v8"
```

Comment:
yeah `validator` is used else where and then conflicts with this import hence the `valid` name ... still thinking